tests/unit/date/helper.js

```diff
@@ -9,10 +9,10 @@ return $.extend( helper, {
 	getAttributes: function( locale ) {
 		var globalize = new Globalize( locale ),
 			weekdayShortFormatter = globalize.dateFormatter( { raw: "EEEEEE" } ),
-			weekdayNarrowFormatter = globalize.dateFormatter( { raw: "EEEEE" } );
+			firstDayRaw = globalize.dateFormatter( { raw: "c" } )( new Date( 1970, 0, 3 ) );
 
 		return {
-			firstDay: globalize.cldr.supplemental.weekData.firstDay(),
+			firstDay: ( 6 - globalize.parseNumber( firstDayRaw ) + 1 ),
```

Comment:
Specifically, the subtraction should cast since there is no `-` operator for strings. If we were adding the two values, then you'd have to explicitly cast.